docs/public-networks/reference/evm-tool.md

```diff
@@ -433,130 +430,118 @@ Displays the version information.
-### Use standard input
-
-If no reference tests are passed in using the command line, the EVM tool loads one complete JSON object from standard input and executes that state test.
-
-<!--tabs-->
-
-# Docker example
+#### Use standard input
 
-```bash
-docker run --rm -i hyperledger/besu-evmtool:develop --json state-test < stExample/add11.json
-```
-
-# CLI example
+If no reference tests are passed in using the command line, the EVM tool loads and validates code
+from standard input.
+Each line is considered a separate program.
+Comment lines and blanks are ignored.
 
-```bash
-evm --json state-test < stExample/add11.json
-```
+### `state-test`
 
-<!--/tabs-->
+Allows the [Ethereum state tests](https://github.com/ethereum/tests/tree/develop/GeneralStateTests)
+to be evaluated.
+The only applicable options are [`--json`](#json-trace) and [`--nomemory`](#nomemory-tracenomemory).
 
-## EOF code validation
+Set `--json` option for EVM Lab Fuzzing.
+Whether or not `--json` is set, a summary JSON object is printed to standard output for each state
+test executed.
 
-The `code-validate` subcommand allows [Ethereum object formatted (EOF)](https://eips.ethereum.org/EIPS/eip-3540) code to be validated. It accepts candidate EOF containers or EVM bytecode using the `--file` option, command arguments, or standard input.
+#### Use command arguments
 
-### `file`
+If you use command arguments, you can list one or more state tests.
+All the state tests are evaluated in the order they are specified.
 
 <!--tabs-->
 
-# Syntax
+# Docker example
 
 ```bash
---file=<file>
+docker run --rm -v ${PWD}:/opt/referencetests hyperledger/besu-evmtool:develop --json state-test /opt/referencetests/GeneralStateTests/stExample/add11.json
 ```
 
-# Example
+# CLI example
 
 ```bash
---file=eof.txt
+evmtool --json state-test stExample/add11.json
 ```
 
 <!--/tabs-->
 
-File containing one or more EOF containers or EVM bytecode. Each line in the file is considered a separate program.
-
-### Use command arguments
+#### Use standard input
 
-If you use command arguments, each argument is considered a separate program. If a code segment includes spaces, it must be contained in quotes.
+If no reference tests are passed in using the command line, the EVM tool loads one complete JSON
+object from standard input and executes that state test.
 
 <!--tabs-->
 
 # Docker example
 
 ```bash
-docker run --rm hyperledger/besu-evmtool:develop code-validate "0xef0001 010008 020002-0007-0002 030000 00 00000002-02010002 59-59-b00001-50-b1 03-b1" 0xef0002 0xef00010100040200010001030000000000000000
+docker run --rm -i hyperledger/besu-evmtool:develop --json state-test < stExample/add11.json
 ```
 
 # CLI example
 
 ```bash
-evm code-validate "0xef0001 010008 020002-0007-0002 030000 00 00000002-02010002 59-59-b00001-50-b1 03-b1" 0xef0002 0xef00010100040200010001030000000000000000
+evmtool --json state-test < stExample/add11.json
 ```
 
 <!--/tabs-->
 
-### Use standard input
+### `transition`, `t8n`
```

Comment:
Might also want to mention the `t8n-server` command which is just t8n but using a continuous input/output stream to run tests rather than only executing one test.  There is no docs for it but https://github.com/ethereum/execution-spec-tests dies reference it.